src/platforms/react-native/manual-setup/native-init.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Native Initialization
+sidebar_order: 4
+description: "Learn how to manually initialize the native SDKs."
+---
+
+By default, the React Native SDK will initialize the native SDK underneath when the `init` method is called on the JS layer. This currently has the limitation of not capturing native crashes that occur before the `init` method is called on the JS layer. You could also look to initialize the native SDKs yourself to be able to provide custom options above what the React Native SDK currently provides.
```

Comment:
It seems like initializing the SDK manually is what you're telling the user to do instead of letting the default initialization process happen, so I tried to make the contrast clearer. 

```suggestion
By default, the React Native SDK will initialize the native SDK underneath when the `init` method is called on the JavaScript layer. However, this method currently has the limitation of not capturing native crashes that occur before the `init` method is called on the JavaScript layer. To overcome this, you can initialize the native SDKs manually to provide custom options beyond what the React Native SDK currently provides.
```

Reply:
Hmm this one is an either or, so you maybe also choose to do this if you want to provide custom options.